Llk/Parser.php

```diff
@@ -192,7 +192,7 @@ public function parse($text, $rule = null, $tree = true)
                 $column = 1;
 
                 if (!empty($text)) {
-                    if (0 === $offset) {
+                    if (0 === $offset || null === $offset) {
```

Comment:
So `$offset` can be set to 0 or `null`. Is it because `$tokens` does not exist or because `$token['offset']`is really equal to `null`?